templates/taxi/driver_list.html

```diff
@@ -1,17 +1,28 @@
 {% extends "base.html" %}
 
 {% block content %}
-  <h1>Driver list</h1>
-  {% if driver_list %}
-    <ul>
-      {% for driver in driver_list%}
-        <li>
-          <a href="{% url "taxi:driver-detail" pk=driver.id %}">{{ driver.username }}</a>
-          ({{ driver.first_name }} {{ driver.last_name }})
-        </li>
-      {% endfor %}
-    </ul>
-  {% else %}
-    <p>There are no drivers in taxi</p>
-  {% endif %}
+<table class="table">
+    <thead>
+        <tr>
+            <th>Username</th>
+            <th>License Number</th>
+        </tr>
+    </thead>
+    <tbody>
+        {% for driver in object_list %}
+        <tr>
+            <td>
+                <a href="{% url 'taxi:driver-detail' driver.id %}">
```

Comment:
The {% url %} template tag generates URLs dynamically using the URL name defined in your urls.py file. The parameters you pass must match the named parameters expected by the URL pattern. 

Example:

`<a href="{% url 'taxi:car-update' car.id %}" class="btn btn-primary btn-sm">Update</a>`

> Since your URL pattern uses <int:pk>, you should use pk= in the template tag:

`<a href="{% url 'taxi:car-update' pk=car.id %}" class="btn btn-primary btn-sm">Update</a>`

Ensure that the href in your template matches the URL pattern, or update the URL pattern to match the parameters used in your template.